src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -4958,7 +4958,14 @@ IOperation popTarget(IOperation originalTarget, ImmutableArray<IOperation> argum
                     case OperationKind.ArrayElementReference:
                         Debug.Assert(((IArrayElementReferenceOperation)originalTarget).Indices.Length == arguments.Length);
                         return new ArrayElementReferenceExpression(instance, arguments, semanticModel: null, originalTarget.Syntax, originalTarget.Type, originalTarget.ConstantValue, IsImplicit(originalTarget));
+                    case OperationKind.DynamicIndexerAccess:
+                        var dynamicAccess = (HasDynamicArgumentsExpression)originalTarget;
+                        Debug.Assert(dynamicAccess.Arguments.Length == arguments.Length);
+                        return new DynamicIndexerAccessExpression(instance, arguments, dynamicAccess.ArgumentNames, dynamicAccess.ArgumentRefKinds, semanticModel: null,
+                                                                  dynamicAccess.Syntax, dynamicAccess.Type, dynamicAccess.ConstantValue, IsImplicit(dynamicAccess));
```

Comment:
It doesn't look like there is a test covering this code path

Reply:
It actually is covered, just hidden. Will push a change shortly that addresses this.

---
In reply to: [193594776](https://github.com/dotnet/roslyn/pull/27519#discussion_r193594776) [](ancestors = 193594776)